crypto/engine/eng_cryptodev.c

```diff
@@ -1247,13 +1247,21 @@ cryptodev_asym(struct crypt_kop *kop, int rlen, BIGNUM *r, int slen,
         return (ret);
 
     if (r) {
-        kop->crk_param[kop->crk_iparams].crp_p = calloc(rlen, sizeof(char));
+        kop->crk_param[kop->crk_iparams].crp_p = OPENSSL_zalloc(rlen);
+        if (kop->crk_param[kop->crk_iparams].crp_p == NULL)
+            return (ret);
```

Comment:
Well, yes, I followed the coding style in the rest of the file. Changed now, and all changes have been pushed.